ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -373,7 +385,25 @@ const CreateRunPage = () => {
         }
       : undefined;
     const evaluatorConfig = values.jobConfigs.evaluator.enabled
-      ? {}
+      ? {
+          // Only include the config param structures if the corresponding form fields are not empty.
+          // In case they are empty, the default path from the config file provider will be used to
+          // resolve the corresponding files.
+          ...(values.jobConfigs.evaluator.copyrightGarbageFile && {
+            copyrightGarbageFile:
+              values.jobConfigs.evaluator.copyrightGarbageFile,
+          }),
+          ...(values.jobConfigs.evaluator.licenseClassificationsFile && {
+            licenseClassificationsFile:
+              values.jobConfigs.evaluator.licenseClassificationsFile,
+          }),
+          ...(values.jobConfigs.evaluator.resolutionsFile && {
+            resolutionsFile: values.jobConfigs.evaluator.resolutionsFile,
+          }),
+          ...(values.jobConfigs.evaluator.ruleSet && {
+            ruleSet: values.jobConfigs.evaluator.ruleSet,
+          }),
```

Comment:
I don't think there's need to spread here, I beliebe the key can exist if `undefined` is given. This could be a little simplere like

```ts
copyrightGarbageFile: values.jobConfigs.evaluator.copyrightGarbageFile || undefined
```

Reply:
Already tried, didn't work.

Reply:
Alright, probably a bug in the API then.

Reply:
I tried this with `copyrightGarbageFile: values.jobConfigs.evaluator.copyrightGarbageFile || undefined` and it did work for me. How did you get it to not work?

Reply:
Try evaluation rules file.

Reply:
That was just for example, the code looks like:

```ts
    const evaluatorConfig = values.jobConfigs.evaluator.enabled
      ? {
          // Only include the config param structures if the corresponding form fields are not empty.
          // In case they are empty, the default path from the config file provider will be used to
          // resolve the corresponding files.
            copyrightGarbageFile:
              values.jobConfigs.evaluator.copyrightGarbageFile || undefined,
            licenseClassificationsFile:
              values.jobConfigs.evaluator.licenseClassificationsFile || undefined,
            resolutionsFile: values.jobConfigs.evaluator.resolutionsFile || undefined,
            ruleSet: values.jobConfigs.evaluator.ruleSet || undefined,
        }
      : undefined;
```

Reply:
OK I must've then been wrong in my tests, I'll make the changes and retest.

Reply:
Done and working with your suggested changes, thanks for verifying this!